lib/origen/errata.rb

```diff
@@ -92,9 +92,7 @@ def _errata
     end
  
     def _sw_workarounds
-      @_sw_workarounds ||= Hash.new do |h,k|
```

Comment:
change this back to being a 2-D hash.  For some reason, the 1-D hash is causing origen specs to fail.